orangecontrib/educational/widgets/owpolynomialregression.py

```diff
@@ -324,13 +373,6 @@ def apply(self):
         self.Outputs.model.send(predictor)
 
         # Send model coefficents
-        model = None
-        if predictor is not None:
```

Comment:
This code is moved to an earlier place and the check is changed to `if hasattr(predictor, "model")`, which covers both `None` and `RegressTo0` (which is new in this code). A bit later, the code expects that `predictor` is not `None`. I have not changed this, so if it was broken before, it still is (though we used this widget a lot and no student ever complained).